__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -2536,3 +2537,32 @@ describe('getTxHistory', () => {
     expect(txHistory.length).toEqual(2);
   });
 });
+
+describe('storage methods', () => {
+  afterEach(async () => {
+    await stopAllWallets();
+    await GenesisWalletHelper.clearListeners();
+  });
+
+  it('should configure the gap limit for the wallet', async () => {
+    const hWallet = await generateWalletHelper();
+    await hWallet.setGapLimit(100);
+    await expect(hWallet.storage.getGapLimit()).resolves.toEqual(100);
```

Comment:
Does it make sense to create a `getGapLimit` in the wallet facade?

Reply:
Done